performance/get_time.js

```diff
@@ -0,0 +1,16 @@
+const path = require('path');
+const { VASTClient } = require('../dist/vast-client-node.min.js');
+
+const url = path.join('file://', __dirname, '..', 'spec', 'samples', 'inline-linear.xml');
```

Comment:
FWIW the output from `file-url` is like `file:///C:/git/vast-client-js/spec/samples/inline-linear.xml` which when loaded in any browser on Windows works just fine.

Unclear to me why node wants to load `C:\C:\`
It is either issue in `vast-client` or the lib which it uses under the hood to issue requests, or some peculiary of msys/mingw like [this one](https://stackoverflow.com/questions/28533664/how-to-prevent-msys-to-convert-the-file-path-for-an-external-program#28536585)

Reply:
I changed it to using `pathtofileurl`. I wasn't able to test it on windows though, so I'm not sure it will work. I also removed the keyword `__dirname` as it wasn't necessary after all